app/models/conference.rb

```diff
@@ -280,8 +280,8 @@ def get_status
   # ====Returns
   # * +hash+ -> user: submissions
   def self.get_top_submitter(limit = 5)
-    submitter = EventUser.where('event_role = ?', 'submitter').limit(limit).group(:user_id)
-    counter = submitter.order('count_all desc').count
+    submitter = EventUser.select(:user_id).where('event_role = ?', 'submitter').limit(limit).group(:user_id)
```

Comment:
@ChrisBr I think you had initially written this, right? Could you please review these changes? On mysql they seem to return different results, but I am not sure if that affects the end result we are getting. I am also a bit confused about the result counter gives us.